src/app/items/containers/item-edit-wrapper/item-edit-wrapper.component.ts

```diff
@@ -140,6 +153,28 @@ export class ItemEditWrapperComponent implements OnInit, OnChanges, OnDestroy, P
     return this.itemForm.dirty;
   }
 
+  private fetchOtherLanguages(mainItem: Item): void {
+    if (this.fetchingOtherLanguages()) return;
+    const languagesForFetch = this.supportedLanguages().filter(langTag => langTag !== mainItem.string.languageTag);
+    if (languagesForFetch.length > 0) {
+      this.itemForm.disable();
+      this.fetchingOtherLanguages.set(true);
+      forkJoin(languagesForFetch.map(langTag =>
+        this.getItemByIdService.get(mainItem.id, { languageTag: langTag })
+      )).subscribe({
+        next: result => this.updateAllStringsFormValue(result),
+        error: () => {
+          this.fetchingOtherLanguages.set(false);
+          this.itemForm.disable();
+        },
+        complete: () => {
+          this.fetchingOtherLanguages.set(false);
+          this.itemForm.disable();
```

Comment:
`disable` for completion and `disabled` before fetching? what is `disable` for ?